### PR TITLE
Update angular 9->13

### PR DESCRIPTION
**Pull request by maltaesousa**
This PR updates front libraries including Angular. Front-end user tests would need to be done before deploying.

### CODE REVIEW

**Comment by maltaesousa** [comment]
@kalbermattenm a nice review would be if you can run it on your dev environment :)

File: README.md [hunk 1]
```diff
@@ -186,3 +186,29 @@ Create an `env.prod` file base on `env.sample`.
 Create a scheduled task that runs `scripts/geoshop_clean_orders.ps1` every month.
 
 More info on bookstack
+
+
+## Upgading front-end packages
```
Comment (kalbermattenm) [inline README.md]:
```suggestion
## Upgrading front-end packages
```

**Review by kalbermattenm — APPROVED** [review]
Nice work!

**Comment by kalbermattenm** [comment]
I haven't run it yet!

I will try!

**Pull request merged**